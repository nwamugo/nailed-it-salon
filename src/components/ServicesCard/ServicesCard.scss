.services-bouquet {
  width: 70.8rem;
  height: 28.571rem;

  @media (max-width: 26.75em) { 
    width: 100%;
    height: 28.571rem;
  }
}
