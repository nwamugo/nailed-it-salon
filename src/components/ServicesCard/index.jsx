import React from 'react';

import './ServicesCard.scss';
import ServicesDetail from '../ServicesDetail';

const ServicesCard = props => {
  return (
    <div className="services-bouquet">
      <ServicesDetail image={props.image} cardTitle={props.cardTitle} />
    </div>
  );
};

export default ServicesCard;
