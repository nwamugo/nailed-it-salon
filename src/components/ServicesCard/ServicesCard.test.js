import React from 'react';
import { shallow } from 'enzyme';
import ServicesCard from './index';

describe('ServicesCard', () => {
  it('should not have regressed', () => {
    const component = shallow(<ServicesCard />);

    expect(component).toMatchSnapshot();
  });
});
