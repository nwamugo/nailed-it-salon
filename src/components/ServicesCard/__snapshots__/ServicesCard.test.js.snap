// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServicesCard should not have regressed 1`] = `
<div
  className="services-bouquet"
>
  <ServicesDetail />
</div>
`;
